src/charm_state.py

```diff
@@ -372,6 +377,33 @@ def _parse_openstack_clouds_config(cls, charm: CharmBase) -> dict | None:
 
         return cast(dict, openstack_clouds_yaml)
 
+    @validator("reconcile_interval")
+    @classmethod
+    def check_reconcile_interval(cls, reconcile_interval: int) -> int:
+        """Validate the general charm configuration.
+
+        Args:
+            reconcile_interval: The value of reconcile_interval passed to class instantiation.
+
+        Raises:
+            ValueError: if an invalid reconcile_interval value of less than 2 has been passed.
+
+        Returns:
+            The validated reconcile_interval value.
+        """
+        # The EventTimer class sets a timeout of `reconcile_interval` - 1.
+        # Therefore the `reconcile_interval` must be at least 2.
+        if reconcile_interval < 2:
+            logger.exception(
+                "The %s configuration must be greater than 1", RECONCILE_INTERVAL_CONFIG_NAME
+            )
```

Comment:
`Logger.exception` is only useful within exception handlers (`try: ... except: ...`). In other contexts, it may be more fitting to use `Logger.error`.

Reply:
You're right, thanks! Fixed :D